aspnetcore/performance/response-compression/sample/Program.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace ResponseCompressionSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = new WebHostBuilder()

```

Comment:
@JunTaoLuo Removed my last comment. I think it's a separate discussion: https://github.com/aspnet/IISIntegration/issues/318